styles/style.css

```diff
@@ -0,0 +1,1681 @@
+      align-self: center;
+}
+
+@media only screen and (max-width: 500px) {
+  .image {
+    max-width: 33%;
+  }
+}
+
+@media only screen and (max-width: 390px) {
+  .image {
+    max-width: 45%;
+  }
+}
+
+@media only screen and (max-width: 390px) {
+  .image {
+    max-width: 50%;
+  }
+}
+
+.image--rounded {
+  border-radius: 50%;
+}
+
+.input {
+  padding: 0.3rem 0.2rem;
+  border: 1px solid var(--primary-color);
+  outline: 1px solid var(--primary-color);
+  color: var(--primary-color);
+}
+
+.input-label {
+  margin: 0.5rem 0;
+  color: var(--primary-color);
+}
+
+.input-required {
+  margin-top: 0.5rem;
+}
+
+.input-container {
+  max-width: 50%;
+}
+
+@media only screen and (max-width: 390px) {
+  .input-container {
+    max-width: 60%;
+  }
+}
+
+@media only screen and (max-width: 345px) {
+  .input-container {
+    max-width: 70%;
+  }
+}
+
+.input--error {
+  border-color: #DC2626;
+  outline-color: #EF4444;
+}
+
+.input:focus {
+  border-color: var(--secondary-color);
+  outline-color: var(--secondary-color);
+}
+
+.list {
+  text-align: center;
+  -ms-flex-item-align: center;
+      -ms-grid-row-align: center;
+      align-self: center;
+  min-width: 60%;
+}
+
+@media only screen and (max-width: 390px) {
+  .list {
+    min-width: 75%;
+  }
+}
+
+.list-stacked {
+  max-height: 42vh;
+  overflow-y: auto;
+  -webkit-box-shadow: -1px -1px 3px 0 rgba(1, 1, 1, 0.1);
+          box-shadow: -1px -1px 3px 0 rgba(1, 1, 1, 0.1);
+  border: 1px solid #D1D5DB;
+}
+
+.list__item {
+  padding: 1rem;
+  cursor: pointer;
+  border-bottom: 1px solid #D1D5DB;
+}
+
+.list__item--spaced {
+  margin: 0.5rem 0;
+  padding: 1rem;
+  cursor: pointer;
+  border: 1px solid #D1D5DB;
+}
+
+.list__item--spaced:hover {
+  -webkit-box-shadow: 2px 2px 3px 0 rgba(1, 1, 1, 0.1);
+          box-shadow: 2px 2px 3px 0 rgba(1, 1, 1, 0.1);
+  -webkit-transform: scale(1.03);
+          transform: scale(1.03);
+}
+
+.list__item:hover {
+  background-color: #BFDBFE;
+}
+
+.search-bar {
+  max-height: 7vh;
+  max-width: 50%;
+}
+
+@media only screen and (max-width: 500px) {
+  .search-bar {
+    max-width: 60%;
+  }
+}
+
+@media only screen and (max-width: 345px) {
+  .search-bar {
+    max-width: 67%;
+  }
+}
+
+.search-bar__input {
+  width: 100%;
+  padding: 0.4rem;
+  font-size: 1.2rem;
+  color: var(--dark-color);
+  border: 1px solid var(--primary-color);
+  border-radius: 0.3rem;
+}
+
+@media only screen and (max-width: 500px) {
+  .search-bar__input {
+    font-size: 1rem;
+  }
+}
+
+.search-bar__input:focus {
+  outline: thin dotted var(--secondary-color);
+  border-color: var(--secondary-color);
+}
+
+.search-bar__icon {
+  cursor: pointer;
+  position: absolute;
+  top: 0.5rem;
+  right: 0.5rem;
+  font-size: 1.2rem;
+  color: var(--secondary-color);
+}
+
+@media only screen and (max-width: 500px) {
+  .search-bar__icon {
+    top: 0.4rem;
+  }
+}
+
+.modal {
+  position: fixed;
+  width: 100vw;
+  height: 100vh;
+  top: 0;
+  left: 0;
+  z-index: 100;
```

Comment:
since most of the the z index needs can be satisfied under 10. it would be a good practice to not use big z index values without absolutely required